src/bindings/python/tests/test_transformations/test_pattern_ops.py

```diff
@@ -20,6 +20,165 @@
 
 from tests.test_transformations.utils.utils import expect_exception
 
+def test_simple_model_and_pattern():
+    # Create a sample model
+    model_param1 = ops.parameter(PartialShape([2, 2]))
```

Comment:
Sample model looks pretty the same in many tests. Let's move it to separate function.

Reply:
Actually, it's not the same and there're differences in almost each test case.